src/book/03-guides/01-main-codebase/04-robocup-setup.mdx

```diff
@@ -0,0 +1,34 @@
+---
+section: Guides
+chapter: Main Codebase
+title: Robocup Setup
+description: Setting up the robot for robocup
+slug: /guides/main/robocup-setup
+authors:
+    - Cameron Murtagh (@CMurtagh-LGTM)
+---
+
+## Networking
+
+1. On the robot with a monitor and keyboard, not via `ssh`.
```

Comment:
Maybe have this as an Alert as it's not really a step but more a method of operation
```suggestion
<Alert type='info'>

Network setup must be completed on the robot with a monitor and keyboard, not via `ssh`.

</Alert>
```

Reply:
If you do this right you can do it by ssh. We always used to. Only had to go to mouse and keyboard when we screwed up :P